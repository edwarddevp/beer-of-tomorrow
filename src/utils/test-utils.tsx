import React, { FC, ReactElement } from "react";
import { render, RenderOptions } from "@testing-library/react";
import { ChakraProvider } from "@chakra-ui/react";
import theme from "../theme";
import { createSerializer, matchers } from "@emotion/jest";
import { useStore } from "@/configs/store";
import { Provider } from "react-redux";
import mockBeerItems from "./mock-beer-items";
// @ts-ignore
expect.addSnapshotSerializer(createSerializer());
// @ts-ignore
expect.extend(matchers);

const initialStoreValues = {
  getBeerById: {
    data: mockBeerItems[0],
    isLoading: false,
    errors: [],
  },
};

const AllTheProviders: FC = ({ children }) => {
  const store = useStore(initialStoreValues);
  return (
    <ChakraProvider theme={theme}>
      <Provider store={store}>{children}</Provider>
    </ChakraProvider>
  );
};

const customRender = (
  ui: ReactElement,
  options?: Omit<RenderOptions, "wrapper">
) => render(ui, { wrapper: AllTheProviders, ...options });

// re-export everything
export * from "@testing-library/react";

// override render method
export { customRender as render };
