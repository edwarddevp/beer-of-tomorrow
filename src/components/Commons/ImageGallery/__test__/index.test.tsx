import React from "react";
// Using render and screen from test-utils.js instead of
// @testing-library/react
import { fireEvent, render } from "test-utils";
import { ImageGallery } from "../";

describe("Image Gallery", () => {
  it("renders", () => {
    const { container } = render(<ImageGallery />);
    expect(container).toMatchSnapshot();
  });
  it("renders aside Elements", () => {
    const { getByAltText } = render(<ImageGallery />);
    expect(getByAltText("Beer-Image-0")).toBeTruthy();
  });
  it("Aside elements events", () => {
    const { getByAltText, container } = render(<ImageGallery />);
    const image = getByAltText("Beer-Image-0");
    fireEvent.mouseOver(image);
    fireEvent.click(image);
    fireEvent.mouseLeave(image);
    expect(container).toMatchSnapshot();
  });
});
