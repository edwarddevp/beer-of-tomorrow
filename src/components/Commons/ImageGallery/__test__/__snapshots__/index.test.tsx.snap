// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Image Gallery Aside elements events 1`] = `
.emotion-0 {
  display: grid;
  grid-column: 1;
  grid-row: 1;
  padding-top: var(--chakra-space-4);
  padding-bottom: var(--chakra-space-4);
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
}

@media screen and (min-width: 62em) {
  .emotion-0 {
    grid-column: 2;
    padding-top: var(--chakra-space-16);
    padding-bottom: var(--chakra-space-16);
  }
}

.emotion-1 {
  display: grid;
  grid-gap: var(--chakra-space-2);
  place-items: center;
  padding-top: var(--chakra-space-16);
  padding-bottom: var(--chakra-space-8);
  text-align: center;
}

@media screen and (min-width: 30em) {
  .emotion-1 {
    display: grid;
  }
}

@media screen and (min-width: 48em) {
  .emotion-1 {
    display: grid;
  }
}

@media screen and (min-width: 62em) {
  .emotion-1 {
    display: none;
  }
}

.emotion-2 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: center;
  -ms-flex-pack: center;
  -webkit-justify-content: center;
  justify-content: center;
  -webkit-flex-direction: row;
  -ms-flex-direction: row;
  flex-direction: row;
  color: var(--chakra-colors-gray-600);
  font-size: var(--chakra-fontSizes-md);
}

.emotion-2>*:not(style)~*:not(style) {
  margin-top: 0px;
  -webkit-margin-end: 0px;
  margin-inline-end: 0px;
  margin-bottom: 0px;
  -webkit-margin-start: 0.5rem;
  margin-inline-start: 0.5rem;
}

@media screen and (min-width: 62em) {
  .emotion-2 {
    font-size: var(--chakra-fontSizes-lg);
  }
}

.emotion-3 {
  transition-property: var(--chakra-transition-property-common);
  transition-duration: var(--chakra-transition-duration-fast);
  transition-timing-function: var(--chakra-transition-easing-ease-out);
  cursor: pointer;
  -webkit-text-decoration: none;
  text-decoration: none;
  outline: 2px solid transparent;
  outline-offset: 2px;
  color: inherit;
}

.emotion-3:hover,
.emotion-3[data-hover] {
  -webkit-text-decoration: none;
  text-decoration: none;
  color: var(--chakra-colors-primary-500);
}

.emotion-3:focus,
.emotion-3[data-focus] {
  box-shadow: var(--chakra-shadows-none);
}

.emotion-3:active,
.emotion-3[data-active] {
  box-shadow: var(--chakra-shadows-none);
}

.emotion-4 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-flex-direction: row;
  -ms-flex-direction: row;
  flex-direction: row;
}

.emotion-4>*:not(style)~*:not(style) {
  margin-top: 0px;
  -webkit-margin-end: 0px;
  margin-inline-end: 0px;
  margin-bottom: 0px;
  -webkit-margin-start: 0.5rem;
  margin-inline-start: 0.5rem;
}

.emotion-5 {
  border-bottom-width: 1px;
  padding-bottom: .1rem;
}

.parent-link:hover .emotion-5 {
  border-color: var(--chakra-colors-primary-500);
}

.emotion-6 {
  width: var(--chakra-sizes-2);
  height: var(--chakra-sizes-2);
  -webkit-transform: rotate(45deg);
  -moz-transform: rotate(45deg);
  -ms-transform: rotate(45deg);
  transform: rotate(45deg);
  background: var(--chakra-colors-gray-500);
}

.emotion-8 {
  font-family: var(--chakra-fonts-heading);
  font-weight: var(--chakra-fontWeights-bold);
  font-size: var(--chakra-fontSizes-3xl);
  line-height: 1.33;
}

@media screen and (min-width: 48em) {
  .emotion-8 {
    font-size: var(--chakra-fontSizes-4xl);
    line-height: 1.2;
  }
}

.emotion-9 {
  font-family: var(--chakra-fonts-heading);
  font-weight: var(--chakra-fontWeights-bold);
  font-size: var(--chakra-fontSizes-xl);
  line-height: 1.2;
  color: var(--chakra-colors-gray-500);
}

.emotion-10 {
  display: grid;
  grid-template-columns: 2fr 1fr;
  justify-items: center;
}

.emotion-11 {
  display: grid;
  grid-gap: var(--chakra-space-8);
  margin-top: var(--chakra-space-16);
  margin-bottom: var(--chakra-space-16);
  height: 80vh;
  padding-right: var(--chakra-space-8);
  -webkit-align-content: start;
  -ms-flex-line-pack: start;
  align-content: start;
  overflow: auto;
}

.emotion-12 {
  cursor: pointer;
  position: relative;
}

<div>
  <div
    class="emotion-0"
  >
    <div
      class="emotion-1"
    >
      <div
        class="chakra-stack emotion-2"
      >
        <a
          class="chakra-link parent-link emotion-3"
          href="#"
        >
          <div
            class="chakra-stack emotion-4"
          >
            <span
              class="emotion-5"
            >
              Lorem
            </span>
          </div>
        </a>
        <div
          class="emotion-6"
        />
        <h3
          class="chakra-text emotion-7"
        >
          Beer of Tomorrow
        </h3>
      </div>
      <h1
        class="chakra-heading emotion-8"
      >
        Buzz
      </h1>
      <h3
        class="chakra-heading emotion-9"
      >
        Venta Distributions
      </h3>
    </div>
    <div
      class="emotion-10"
    >
      <div
        style="display: inline-block; max-width: 100%; overflow: hidden; position: relative; box-sizing: border-box; margin: 0px;"
      >
        <div
          style="box-sizing: border-box; display: block; max-width: 100%;"
        >
          <img
            alt=""
            aria-hidden="true"
            src="data:image/svg+xml;base64,PHN2ZyB3aWR0aD0iMzAwIiBoZWlnaHQ9IjgwMCIgeG1sbnM9Imh0dHA6Ly93d3cudzMub3JnLzIwMDAvc3ZnIiB2ZXJzaW9uPSIxLjEiLz4="
            style="max-width: 100%; display: block; margin: 0px; padding: 0px;"
          />
        </div>
        <img
          alt="Beer Showing Image"
          data-nimg="true"
          decoding="async"
          src="/_next/image?url=https%3A%2F%2Fimages.punkapi.com%2Fv2%2Fkeg.png&w=640&q=75"
          srcset="/_next/image?url=https%3A%2F%2Fimages.punkapi.com%2Fv2%2Fkeg.png&w=384&q=75 1x, /_next/image?url=https%3A%2F%2Fimages.punkapi.com%2Fv2%2Fkeg.png&w=640&q=75 2x"
          style="position: absolute; top: 0px; left: 0px; bottom: 0px; right: 0px; box-sizing: border-box; padding: 0px; margin: auto; display: block; width: 0px; height: 0px; min-width: 100%; max-width: 100%; min-height: 100%; max-height: 100%;"
        />
      </div>
      <div
        class="scrollbar emotion-11"
      >
        <div
          class="emotion-12"
        >
          <div
            style="display: inline-block; max-width: 100%; overflow: hidden; position: relative; box-sizing: border-box; margin: 0px;"
          >
            <div
              style="box-sizing: border-box; display: block; max-width: 100%;"
            >
              <img
                alt=""
                aria-hidden="true"
                src="data:image/svg+xml;base64,PHN2ZyB3aWR0aD0iNDAiIGhlaWdodD0iMTIwIiB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHZlcnNpb249IjEuMSIvPg=="
                style="max-width: 100%; display: block; margin: 0px; padding: 0px;"
              />
            </div>
            <img
              alt="Beer-Image-0"
              data-nimg="true"
              decoding="async"
              src="/_next/image?url=https%3A%2F%2Fimages.punkapi.com%2Fv2%2Fkeg.png&w=96&q=75"
              srcset="/_next/image?url=https%3A%2F%2Fimages.punkapi.com%2Fv2%2Fkeg.png&w=48&q=75 1x, /_next/image?url=https%3A%2F%2Fimages.punkapi.com%2Fv2%2Fkeg.png&w=96&q=75 2x"
              style="position: absolute; top: 0px; left: 0px; bottom: 0px; right: 0px; box-sizing: border-box; padding: 0px; margin: auto; display: block; width: 0px; height: 0px; min-width: 100%; max-width: 100%; min-height: 100%; max-height: 100%;"
            />
          </div>
        </div>
        <div
          class="emotion-12"
        >
          <div
            style="display: inline-block; max-width: 100%; overflow: hidden; position: relative; box-sizing: border-box; margin: 0px;"
          >
            <div
              style="box-sizing: border-box; display: block; max-width: 100%;"
            >
              <img
                alt=""
                aria-hidden="true"
                src="data:image/svg+xml;base64,PHN2ZyB3aWR0aD0iNDAiIGhlaWdodD0iMTIwIiB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHZlcnNpb249IjEuMSIvPg=="
                style="max-width: 100%; display: block; margin: 0px; padding: 0px;"
              />
            </div>
            <img
              alt="Beer-Image-1"
              data-nimg="true"
              decoding="async"
              src="/_next/image?url=https%3A%2F%2Fimages.punkapi.com%2Fv2%2Fkeg.png&w=96&q=75"
              srcset="/_next/image?url=https%3A%2F%2Fimages.punkapi.com%2Fv2%2Fkeg.png&w=48&q=75 1x, /_next/image?url=https%3A%2F%2Fimages.punkapi.com%2Fv2%2Fkeg.png&w=96&q=75 2x"
              style="position: absolute; top: 0px; left: 0px; bottom: 0px; right: 0px; box-sizing: border-box; padding: 0px; margin: auto; display: block; width: 0px; height: 0px; min-width: 100%; max-width: 100%; min-height: 100%; max-height: 100%;"
            />
          </div>
        </div>
        <div
          class="emotion-12"
        >
          <div
            style="display: inline-block; max-width: 100%; overflow: hidden; position: relative; box-sizing: border-box; margin: 0px;"
          >
            <div
              style="box-sizing: border-box; display: block; max-width: 100%;"
            >
              <img
                alt=""
                aria-hidden="true"
                src="data:image/svg+xml;base64,PHN2ZyB3aWR0aD0iNDAiIGhlaWdodD0iMTIwIiB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHZlcnNpb249IjEuMSIvPg=="
                style="max-width: 100%; display: block; margin: 0px; padding: 0px;"
              />
            </div>
            <img
              alt="Beer-Image-2"
              data-nimg="true"
              decoding="async"
              src="/_next/image?url=https%3A%2F%2Fimages.punkapi.com%2Fv2%2Fkeg.png&w=96&q=75"
              srcset="/_next/image?url=https%3A%2F%2Fimages.punkapi.com%2Fv2%2Fkeg.png&w=48&q=75 1x, /_next/image?url=https%3A%2F%2Fimages.punkapi.com%2Fv2%2Fkeg.png&w=96&q=75 2x"
              style="position: absolute; top: 0px; left: 0px; bottom: 0px; right: 0px; box-sizing: border-box; padding: 0px; margin: auto; display: block; width: 0px; height: 0px; min-width: 100%; max-width: 100%; min-height: 100%; max-height: 100%;"
            />
          </div>
        </div>
        <div
          class="emotion-12"
        >
          <div
            style="display: inline-block; max-width: 100%; overflow: hidden; position: relative; box-sizing: border-box; margin: 0px;"
          >
            <div
              style="box-sizing: border-box; display: block; max-width: 100%;"
            >
              <img
                alt=""
                aria-hidden="true"
                src="data:image/svg+xml;base64,PHN2ZyB3aWR0aD0iNDAiIGhlaWdodD0iMTIwIiB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHZlcnNpb249IjEuMSIvPg=="
                style="max-width: 100%; display: block; margin: 0px; padding: 0px;"
              />
            </div>
            <noscript />
            <img
              alt="Beer-Image-3"
              data-nimg="true"
              decoding="async"
              src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
              style="position: absolute; top: 0px; left: 0px; bottom: 0px; right: 0px; box-sizing: border-box; padding: 0px; margin: auto; display: block; width: 0px; height: 0px; min-width: 100%; max-width: 100%; min-height: 100%; max-height: 100%;"
            />
          </div>
        </div>
        <div
          class="emotion-12"
        >
          <div
            style="display: inline-block; max-width: 100%; overflow: hidden; position: relative; box-sizing: border-box; margin: 0px;"
          >
            <div
              style="box-sizing: border-box; display: block; max-width: 100%;"
            >
              <img
                alt=""
                aria-hidden="true"
                src="data:image/svg+xml;base64,PHN2ZyB3aWR0aD0iNDAiIGhlaWdodD0iMTIwIiB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHZlcnNpb249IjEuMSIvPg=="
                style="max-width: 100%; display: block; margin: 0px; padding: 0px;"
              />
            </div>
            <noscript />
            <img
              alt="Beer-Image-4"
              data-nimg="true"
              decoding="async"
              src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
              style="position: absolute; top: 0px; left: 0px; bottom: 0px; right: 0px; box-sizing: border-box; padding: 0px; margin: auto; display: block; width: 0px; height: 0px; min-width: 100%; max-width: 100%; min-height: 100%; max-height: 100%;"
            />
          </div>
        </div>
        <div
          class="emotion-12"
        >
          <div
            style="display: inline-block; max-width: 100%; overflow: hidden; position: relative; box-sizing: border-box; margin: 0px;"
          >
            <div
              style="box-sizing: border-box; display: block; max-width: 100%;"
            >
              <img
                alt=""
                aria-hidden="true"
                src="data:image/svg+xml;base64,PHN2ZyB3aWR0aD0iNDAiIGhlaWdodD0iMTIwIiB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHZlcnNpb249IjEuMSIvPg=="
                style="max-width: 100%; display: block; margin: 0px; padding: 0px;"
              />
            </div>
            <noscript />
            <img
              alt="Beer-Image-5"
              data-nimg="true"
              decoding="async"
              src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
              style="position: absolute; top: 0px; left: 0px; bottom: 0px; right: 0px; box-sizing: border-box; padding: 0px; margin: auto; display: block; width: 0px; height: 0px; min-width: 100%; max-width: 100%; min-height: 100%; max-height: 100%;"
            />
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;

exports[`Image Gallery renders 1`] = `
.emotion-0 {
  display: grid;
  grid-column: 1;
  grid-row: 1;
  padding-top: var(--chakra-space-4);
  padding-bottom: var(--chakra-space-4);
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
}

@media screen and (min-width: 62em) {
  .emotion-0 {
    grid-column: 2;
    padding-top: var(--chakra-space-16);
    padding-bottom: var(--chakra-space-16);
  }
}

.emotion-1 {
  display: grid;
  grid-gap: var(--chakra-space-2);
  place-items: center;
  padding-top: var(--chakra-space-16);
  padding-bottom: var(--chakra-space-8);
  text-align: center;
}

@media screen and (min-width: 30em) {
  .emotion-1 {
    display: grid;
  }
}

@media screen and (min-width: 48em) {
  .emotion-1 {
    display: grid;
  }
}

@media screen and (min-width: 62em) {
  .emotion-1 {
    display: none;
  }
}

.emotion-2 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: center;
  -ms-flex-pack: center;
  -webkit-justify-content: center;
  justify-content: center;
  -webkit-flex-direction: row;
  -ms-flex-direction: row;
  flex-direction: row;
  color: var(--chakra-colors-gray-600);
  font-size: var(--chakra-fontSizes-md);
}

.emotion-2>*:not(style)~*:not(style) {
  margin-top: 0px;
  -webkit-margin-end: 0px;
  margin-inline-end: 0px;
  margin-bottom: 0px;
  -webkit-margin-start: 0.5rem;
  margin-inline-start: 0.5rem;
}

@media screen and (min-width: 62em) {
  .emotion-2 {
    font-size: var(--chakra-fontSizes-lg);
  }
}

.emotion-3 {
  transition-property: var(--chakra-transition-property-common);
  transition-duration: var(--chakra-transition-duration-fast);
  transition-timing-function: var(--chakra-transition-easing-ease-out);
  cursor: pointer;
  -webkit-text-decoration: none;
  text-decoration: none;
  outline: 2px solid transparent;
  outline-offset: 2px;
  color: inherit;
}

.emotion-3:hover,
.emotion-3[data-hover] {
  -webkit-text-decoration: none;
  text-decoration: none;
  color: var(--chakra-colors-primary-500);
}

.emotion-3:focus,
.emotion-3[data-focus] {
  box-shadow: var(--chakra-shadows-none);
}

.emotion-3:active,
.emotion-3[data-active] {
  box-shadow: var(--chakra-shadows-none);
}

.emotion-4 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-flex-direction: row;
  -ms-flex-direction: row;
  flex-direction: row;
}

.emotion-4>*:not(style)~*:not(style) {
  margin-top: 0px;
  -webkit-margin-end: 0px;
  margin-inline-end: 0px;
  margin-bottom: 0px;
  -webkit-margin-start: 0.5rem;
  margin-inline-start: 0.5rem;
}

.emotion-5 {
  border-bottom-width: 1px;
  padding-bottom: .1rem;
}

.parent-link:hover .emotion-5 {
  border-color: var(--chakra-colors-primary-500);
}

.emotion-6 {
  width: var(--chakra-sizes-2);
  height: var(--chakra-sizes-2);
  -webkit-transform: rotate(45deg);
  -moz-transform: rotate(45deg);
  -ms-transform: rotate(45deg);
  transform: rotate(45deg);
  background: var(--chakra-colors-gray-500);
}

.emotion-8 {
  font-family: var(--chakra-fonts-heading);
  font-weight: var(--chakra-fontWeights-bold);
  font-size: var(--chakra-fontSizes-3xl);
  line-height: 1.33;
}

@media screen and (min-width: 48em) {
  .emotion-8 {
    font-size: var(--chakra-fontSizes-4xl);
    line-height: 1.2;
  }
}

.emotion-9 {
  font-family: var(--chakra-fonts-heading);
  font-weight: var(--chakra-fontWeights-bold);
  font-size: var(--chakra-fontSizes-xl);
  line-height: 1.2;
  color: var(--chakra-colors-gray-500);
}

.emotion-10 {
  display: grid;
  grid-template-columns: 2fr 1fr;
  justify-items: center;
}

.emotion-11 {
  display: grid;
  grid-gap: var(--chakra-space-8);
  margin-top: var(--chakra-space-16);
  margin-bottom: var(--chakra-space-16);
  height: 80vh;
  padding-right: var(--chakra-space-8);
  -webkit-align-content: start;
  -ms-flex-line-pack: start;
  align-content: start;
  overflow: auto;
}

.emotion-12 {
  cursor: pointer;
  position: relative;
}

<div>
  <div
    class="emotion-0"
  >
    <div
      class="emotion-1"
    >
      <div
        class="chakra-stack emotion-2"
      >
        <a
          class="chakra-link parent-link emotion-3"
          href="#"
        >
          <div
            class="chakra-stack emotion-4"
          >
            <span
              class="emotion-5"
            >
              Lorem
            </span>
          </div>
        </a>
        <div
          class="emotion-6"
        />
        <h3
          class="chakra-text emotion-7"
        >
          Beer of Tomorrow
        </h3>
      </div>
      <h1
        class="chakra-heading emotion-8"
      >
        Buzz
      </h1>
      <h3
        class="chakra-heading emotion-9"
      >
        Venta Distributions
      </h3>
    </div>
    <div
      class="emotion-10"
    >
      <div
        style="display: inline-block; max-width: 100%; overflow: hidden; position: relative; box-sizing: border-box; margin: 0px;"
      >
        <div
          style="box-sizing: border-box; display: block; max-width: 100%;"
        >
          <img
            alt=""
            aria-hidden="true"
            src="data:image/svg+xml;base64,PHN2ZyB3aWR0aD0iMzAwIiBoZWlnaHQ9IjgwMCIgeG1sbnM9Imh0dHA6Ly93d3cudzMub3JnLzIwMDAvc3ZnIiB2ZXJzaW9uPSIxLjEiLz4="
            style="max-width: 100%; display: block; margin: 0px; padding: 0px;"
          />
        </div>
        <img
          alt="Beer Showing Image"
          data-nimg="true"
          decoding="async"
          src="/_next/image?url=https%3A%2F%2Fimages.punkapi.com%2Fv2%2Fkeg.png&w=640&q=75"
          srcset="/_next/image?url=https%3A%2F%2Fimages.punkapi.com%2Fv2%2Fkeg.png&w=384&q=75 1x, /_next/image?url=https%3A%2F%2Fimages.punkapi.com%2Fv2%2Fkeg.png&w=640&q=75 2x"
          style="position: absolute; top: 0px; left: 0px; bottom: 0px; right: 0px; box-sizing: border-box; padding: 0px; margin: auto; display: block; width: 0px; height: 0px; min-width: 100%; max-width: 100%; min-height: 100%; max-height: 100%;"
        />
      </div>
      <div
        class="scrollbar emotion-11"
      >
        <div
          class="emotion-12"
        >
          <div
            style="display: inline-block; max-width: 100%; overflow: hidden; position: relative; box-sizing: border-box; margin: 0px;"
          >
            <div
              style="box-sizing: border-box; display: block; max-width: 100%;"
            >
              <img
                alt=""
                aria-hidden="true"
                src="data:image/svg+xml;base64,PHN2ZyB3aWR0aD0iNDAiIGhlaWdodD0iMTIwIiB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHZlcnNpb249IjEuMSIvPg=="
                style="max-width: 100%; display: block; margin: 0px; padding: 0px;"
              />
            </div>
            <img
              alt="Beer-Image-0"
              data-nimg="true"
              decoding="async"
              src="/_next/image?url=https%3A%2F%2Fimages.punkapi.com%2Fv2%2Fkeg.png&w=96&q=75"
              srcset="/_next/image?url=https%3A%2F%2Fimages.punkapi.com%2Fv2%2Fkeg.png&w=48&q=75 1x, /_next/image?url=https%3A%2F%2Fimages.punkapi.com%2Fv2%2Fkeg.png&w=96&q=75 2x"
              style="position: absolute; top: 0px; left: 0px; bottom: 0px; right: 0px; box-sizing: border-box; padding: 0px; margin: auto; display: block; width: 0px; height: 0px; min-width: 100%; max-width: 100%; min-height: 100%; max-height: 100%;"
            />
          </div>
        </div>
        <div
          class="emotion-12"
        >
          <div
            style="display: inline-block; max-width: 100%; overflow: hidden; position: relative; box-sizing: border-box; margin: 0px;"
          >
            <div
              style="box-sizing: border-box; display: block; max-width: 100%;"
            >
              <img
                alt=""
                aria-hidden="true"
                src="data:image/svg+xml;base64,PHN2ZyB3aWR0aD0iNDAiIGhlaWdodD0iMTIwIiB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHZlcnNpb249IjEuMSIvPg=="
                style="max-width: 100%; display: block; margin: 0px; padding: 0px;"
              />
            </div>
            <img
              alt="Beer-Image-1"
              data-nimg="true"
              decoding="async"
              src="/_next/image?url=https%3A%2F%2Fimages.punkapi.com%2Fv2%2Fkeg.png&w=96&q=75"
              srcset="/_next/image?url=https%3A%2F%2Fimages.punkapi.com%2Fv2%2Fkeg.png&w=48&q=75 1x, /_next/image?url=https%3A%2F%2Fimages.punkapi.com%2Fv2%2Fkeg.png&w=96&q=75 2x"
              style="position: absolute; top: 0px; left: 0px; bottom: 0px; right: 0px; box-sizing: border-box; padding: 0px; margin: auto; display: block; width: 0px; height: 0px; min-width: 100%; max-width: 100%; min-height: 100%; max-height: 100%;"
            />
          </div>
        </div>
        <div
          class="emotion-12"
        >
          <div
            style="display: inline-block; max-width: 100%; overflow: hidden; position: relative; box-sizing: border-box; margin: 0px;"
          >
            <div
              style="box-sizing: border-box; display: block; max-width: 100%;"
            >
              <img
                alt=""
                aria-hidden="true"
                src="data:image/svg+xml;base64,PHN2ZyB3aWR0aD0iNDAiIGhlaWdodD0iMTIwIiB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHZlcnNpb249IjEuMSIvPg=="
                style="max-width: 100%; display: block; margin: 0px; padding: 0px;"
              />
            </div>
            <img
              alt="Beer-Image-2"
              data-nimg="true"
              decoding="async"
              src="/_next/image?url=https%3A%2F%2Fimages.punkapi.com%2Fv2%2Fkeg.png&w=96&q=75"
              srcset="/_next/image?url=https%3A%2F%2Fimages.punkapi.com%2Fv2%2Fkeg.png&w=48&q=75 1x, /_next/image?url=https%3A%2F%2Fimages.punkapi.com%2Fv2%2Fkeg.png&w=96&q=75 2x"
              style="position: absolute; top: 0px; left: 0px; bottom: 0px; right: 0px; box-sizing: border-box; padding: 0px; margin: auto; display: block; width: 0px; height: 0px; min-width: 100%; max-width: 100%; min-height: 100%; max-height: 100%;"
            />
          </div>
        </div>
        <div
          class="emotion-12"
        >
          <div
            style="display: inline-block; max-width: 100%; overflow: hidden; position: relative; box-sizing: border-box; margin: 0px;"
          >
            <div
              style="box-sizing: border-box; display: block; max-width: 100%;"
            >
              <img
                alt=""
                aria-hidden="true"
                src="data:image/svg+xml;base64,PHN2ZyB3aWR0aD0iNDAiIGhlaWdodD0iMTIwIiB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHZlcnNpb249IjEuMSIvPg=="
                style="max-width: 100%; display: block; margin: 0px; padding: 0px;"
              />
            </div>
            <noscript />
            <img
              alt="Beer-Image-3"
              data-nimg="true"
              decoding="async"
              src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
              style="position: absolute; top: 0px; left: 0px; bottom: 0px; right: 0px; box-sizing: border-box; padding: 0px; margin: auto; display: block; width: 0px; height: 0px; min-width: 100%; max-width: 100%; min-height: 100%; max-height: 100%;"
            />
          </div>
        </div>
        <div
          class="emotion-12"
        >
          <div
            style="display: inline-block; max-width: 100%; overflow: hidden; position: relative; box-sizing: border-box; margin: 0px;"
          >
            <div
              style="box-sizing: border-box; display: block; max-width: 100%;"
            >
              <img
                alt=""
                aria-hidden="true"
                src="data:image/svg+xml;base64,PHN2ZyB3aWR0aD0iNDAiIGhlaWdodD0iMTIwIiB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHZlcnNpb249IjEuMSIvPg=="
                style="max-width: 100%; display: block; margin: 0px; padding: 0px;"
              />
            </div>
            <noscript />
            <img
              alt="Beer-Image-4"
              data-nimg="true"
              decoding="async"
              src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
              style="position: absolute; top: 0px; left: 0px; bottom: 0px; right: 0px; box-sizing: border-box; padding: 0px; margin: auto; display: block; width: 0px; height: 0px; min-width: 100%; max-width: 100%; min-height: 100%; max-height: 100%;"
            />
          </div>
        </div>
        <div
          class="emotion-12"
        >
          <div
            style="display: inline-block; max-width: 100%; overflow: hidden; position: relative; box-sizing: border-box; margin: 0px;"
          >
            <div
              style="box-sizing: border-box; display: block; max-width: 100%;"
            >
              <img
                alt=""
                aria-hidden="true"
                src="data:image/svg+xml;base64,PHN2ZyB3aWR0aD0iNDAiIGhlaWdodD0iMTIwIiB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHZlcnNpb249IjEuMSIvPg=="
                style="max-width: 100%; display: block; margin: 0px; padding: 0px;"
              />
            </div>
            <noscript />
            <img
              alt="Beer-Image-5"
              data-nimg="true"
              decoding="async"
              src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
              style="position: absolute; top: 0px; left: 0px; bottom: 0px; right: 0px; box-sizing: border-box; padding: 0px; margin: auto; display: block; width: 0px; height: 0px; min-width: 100%; max-width: 100%; min-height: 100%; max-height: 100%;"
            />
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
